test/rules/EC2.test.ts

```diff
@@ -536,3 +542,84 @@ describe('Amazon Elastic Block Store (EBS)', () => {
     });
   });
 });
+
+describe('EC2SecurityGroupOnlyTcp443: Security Groups should only allow TCP 443 for ingress iPv4 traffic', () => {
+  const ruleId = 'EC2SecurityGroupOnlyTcp443';
+  test('Noncompliance 1', () => {
+    new CfnSecurityGroup(stack, 'rSecurityGroup', {
+      groupDescription: 'security group tcp port 80 open',
+      securityGroupIngress: [
+        {
+          fromPort: 80,
+          ipProtocol: 'tcp',
+          cidrIp: '0.0.0.0/0',
+        },
+      ],
+    });
+    validateStack(stack, ruleId, TestType.NON_COMPLIANCE);
+  });
+
+  test('Noncompliance 2', () => {
+    new CfnSecurityGroup(stack, 'rSecurityGroup2', {
+      groupDescription: 'security group with udp port 53',
+      securityGroupIngress: [
+        {
+          fromPort: 53,
+          ipProtocol: 'udp',
+          cidrIp: '0.0.0.0/0',
+        },
+      ],
+    });
+    validateStack(stack, ruleId, TestType.NON_COMPLIANCE);
+  });
+
+  test('Compliance', () => {
+    new CfnSecurityGroup(stack, 'rSecurityGroup1', {
+      groupDescription: 'security group with inbound tcp 443',
+      securityGroupIngress: [],
+    });
+    new CfnSecurityGroup(stack, 'rSecurityGroup2', {
+      groupDescription: 'security group with tcp 443 ingress allowed',
+      securityGroupIngress: [
+        {
+          fromPort: 443,
+          ipProtocol: 'tcp',
+          cidrIp: '0.0.0.0/0',
+        },
+      ],
+    });
+    //validateStack(stack, ruleId, TestType.COMPLIANCE)
```

Comment:
This should be uncommented

Reply:
it should be but theres a problem with that rule. But i'm waiting for Craigs comments about this rule before i expend any more effort on it.

Reply:
This is now resolved. additional tests were created.